src/main/java/org/team1540/robot2025/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,76 @@
+package org.team1540.robot2025.subsystems.intake;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.team1540.robot2025.Constants;
+
+public class IntakeIOReal implements IntakeIO {
+
+    // rotation of horizontal beams for the intake, clockwise to intake, counter-clockwise to spit out
+    private final TalonFX topFalcon = new TalonFX(Constants.IntakeConstants.TOP_FALCON_ID);
+
+    // controls height of intake
+    private final TalonFX bottomFalcon = new TalonFX(Constants.IntakeConstants.BOTTOM_FALCON_ID);
+
+    private final PositionVoltage motorRequest = new PositionVoltage(0).withSlot(0);
+
+    // clockwise to intake, counter-clockwise to spit out
+    private final SparkMax neo = new SparkMax(Constants.IntakeConstants.NEO_ID, SparkLowLevel.MotorType.kBrushless);
+
+    public IntakeIOReal() {
+
+        var talonFXConfigs = new TalonFXConfiguration();
+        var slot0configs = talonFXConfigs.Slot0;
+
+        slot0configs.kS = Constants.IntakeConstants.GRABBER_KS;
+        slot0configs.kV = Constants.IntakeConstants.GRABBER_KV;
+        slot0configs.kA = Constants.IntakeConstants.GRABBER_KA;
+        slot0configs.kP = Constants.IntakeConstants.GRABBER_KP;
+        slot0configs.kI = Constants.IntakeConstants.GRABBER_KI;
+        slot0configs.kD = Constants.IntakeConstants.GRABBER_KD;
+
+        var motionMagicConfigs = talonFXConfigs.MotionMagic;
+
+        motionMagicConfigs.MotionMagicCruiseVelocity = Constants.IntakeConstants.GRABBER_CRUISE_VELOCITY;
+        motionMagicConfigs.MotionMagicAcceleration = Constants.IntakeConstants.GRABBER_ACCELERATION;
+
+        bottomFalcon.getConfigurator().apply(slot0configs);
+
+        //        topFalcon.setInverted(false);
+        //        bottomFalcon.setInverted(true);
+        //        neo.setInverted(false);
+
```

Comment:
You will want current limits, and you will want to configure the second motor (and maybe the neo)